README.md

```diff
@@ -27,7 +27,7 @@ Minimum Litestar Implementation.
 
 `$ poetry install`
 
-`$ litestar run --reload`
+`$ poetry run litestar run -r`
```

Comment:
Why?

Reply:
because it is not running for me with `litestar run --reload` after poetry install

Reply:
is it not ok?

Reply:
Just leaving this out here for other reviewers to verify.

> because it is not running for me with `litestar run --reload` after poetry install

So I ran the commands in order, and the order seems wrong? The current order of commands gives me this exact error. I changed the order. `poetry install`, `poetry shell` and `litestar run --reload`, now it works fine.

Reply:
yes  as suggested by @Alc-Alc  i tried that order , it get me to work that way , so we should update doc on that.